src/main/java/edu/wpi/first/pathweaver/CreateProjectController.java

```diff
@@ -0,0 +1,77 @@
+package edu.wpi.first.pathweaver;
+
+import java.io.File;
+import java.util.List;
+
+import javafx.fxml.FXML;
+import javafx.scene.control.Button;
+import javafx.scene.control.TextField;
+import javafx.scene.layout.VBox;
+import javafx.stage.DirectoryChooser;
+
+public class CreateProjectController {
+
+  @FXML
+  private VBox vBox;
+  @FXML
+  private TextField directory;
+  @FXML
+  private Button browse;
+  @FXML
+  private TextField timeStep;
+  @FXML
+  private TextField maxVelocity;
+  @FXML
+  private TextField maxAcceleration;
+  @FXML
+  private TextField maxJerk;
+  @FXML
+  private TextField wheelBase;
+  @FXML
+  private Button create;
+
+  private List<TextField> numericFields;
+
+  @FXML
+  private void initialize() {
+    numericFields = List.of(timeStep, maxVelocity, maxAcceleration, maxJerk, wheelBase);
+    // TODO: validate numbers only
```

Comment:
Numbers can be validated by adding a filter to the text field, for example:

```java
private static final UnaryOperator<TextFormatter.Change> onlyIntegerText = c -> {
  if (c.getControlNewText().matches("^|\\d+$")) {
    return c;
  } else {
    return null;
  }
};

numberFields.forEach(field -> field.setTextFormatter(new TextFormatter<>(onlyIntegerText)));
```

If you want to be able to use decimal points, Shuffleboard has some classes for that:
[AbstractNumberField](https://github.com/wpilibsuite/shuffleboard/blob/master/api/src/main/java/edu/wpi/first/shuffleboard/api/components/AbstractNumberField.java)
[NumberField](https://github.com/wpilibsuite/shuffleboard/blob/master/api/src/main/java/edu/wpi/first/shuffleboard/api/components/NumberField.java)
[IntegerField](https://github.com/wpilibsuite/shuffleboard/blob/master/api/src/main/java/edu/wpi/first/shuffleboard/api/components/IntegerField.java)